fatiando/mesher/mesh.py

```diff
@@ -291,11 +280,89 @@ def addprop(self, prop, values):
         * prop : str
             Name of the physical property.
         * values :  list or array
-            Value of this physical property in each point of the grid
+            Value of this physical property in each point of the mesh.
 
         """
         self.props[prop] = values
 
+    def copy(self):
+        """ Return a deep copy of the current instance."""
+        return cp.deepcopy(self)
+
+
+class PointGrid(IrregularPointMesh):
+    """
+    A regular grid of 3D point sources (spheres of unit volume).
+
+    Use this as a 1D list of :class:`~fatiando.mesher.Sphere`.
+
+    Grid points are ordered like a C matrix, first each row in a column, then
+    change columns. In this case, the x direction (North-South) are the rows
+    and y (East-West) are the columns.
+
+    Parameters:
+
+    * area : list = [x1, x2, y1, y2]
+        The area where the grid will be spread out
+    * z : float or 1d-array
+        The z coordinates of each point in the grid (remember, z is positive
+        downward).
+    * shape : tuple = (nx, ny)
+        The number of points in the x and y directions
+    * props :  dict
+        Physical properties of each point in the grid.
+        Each key should be the name of a physical property. The corresponding
+        value should be a list with the values of that particular property for
+        each point in the grid.
+
+    Examples::
+
+        >>> g = PointGrid([0, 10, 2, 6], 200, (2, 3))
+        >>> g.shape
+        (2, 3)
+        >>> g.size
+        6
+        >>> g[0].center
+        array([   0.,    2.,  200.])
+        >>> g[-1].center
+        array([  10.,    6.,  200.])
+        >>> for p in g:
+        ...     p.center
+        array([   0.,    2.,  200.])
+        array([   0.,    4.,  200.])
+        array([   0.,    6.,  200.])
+        array([  10.,    2.,  200.])
+        array([  10.,    4.,  200.])
+        array([  10.,    6.,  200.])
+        >>> g.x
+        array([  0.,   0.,   0.,  10.,  10.,  10.])
+        >>> g.x.reshape(g.shape)
+        array([[  0.,   0.,   0.],
+               [ 10.,  10.,  10.]])
+        >>> g.y.reshape(g.shape)
+        array([[ 2.,  4.,  6.],
+               [ 2.,  4.,  6.]])
+        >>> g.dx, g.dy
+        (10.0, 2.0)
+
+    """
+
+    def __init__(self, area, z, shape, props=None):
+        self.area = area
+        self.shape = shape
+        if props is None:
+            self.props = {}
+        else:
+            self.props = props
+        nx, ny = shape
+        self.size = nx*ny
+        z = np.zeros(self.size) + z
+        self.radius = scipy.special.cbrt(3. / (4. * np.pi))
+        x, y = gridder.regular(area, shape)
+        # The spacing between points
+        self.dx, self.dy = gridder.spacing(area, shape)
+        super().__init__(x, y, z, props)
```

Comment:
Any attributes set by `IrregularPointMesh` don't need to be set here as well (`props`, `radius`, `size`).